%% start of file `template.tex'.
%% Copyright 2006-2013 Xavier Danaux (xdanaux@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.


\documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')

% modern themes
\moderncvstyle{banking}                            % style options are 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{blue}                                % color options 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'
%\renewcommand{\familydefault}{\sfdefault}         % to set the default font; use '\sfdefault' for the default sans serif font, '\rmdefault' for the default roman one, or any tex font name
%\nopagenumbers{}                                  % uncomment to suppress automatic page numbering for CVs longer than one page

% character encoding
\usepackage[utf8]{inputenc}                       % if you are not using xelatex ou lualatex, replace by the encoding you are using
%\usepackage{CJKutf8}                              % if you need to use CJK to typeset your resume in Chinese, Japanese or Korean

% adjust the page margins
\usepackage[scale=0.94]{geometry}
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvtitlenamewidth}{10cm}           % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...

\usepackage{import}

% personal data
\name{Andrew C. Patt}{}
\phone[mobile]{(518) 366 4293}                   % optional, remove / comment the line if not wanted
\email{patt.14@osu.edu}                               % optional, remove / comment the line if not wanted
\homepage{andyptt21.github.io}

 % optional, remove / comment the line if not wanted
%\extrainfo{github.com/andyptt21}                 % optional, remove / comment the line if not wanted
%\photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file
%\quote{Some quote}                                 % optional, remove / comment the line if not wanted

% to show numerical labels in the bibliography (default is to show no labels); only useful if you make citations in your resume
%\makeatletter
%\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
%\makeatother
%\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}% CONSIDER REPLACING THE ABOVE BY THIS

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}
%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle

\section{Education}

\vspace{0pt}

\begin{itemize}

\item{\cventry{Columbus, OH}{PhD in Biomedical Science, concentration in Biomedical Informatics}{The Ohio State University}{Expected Graduation: March 2021}{\textit{GPA: 3.813/4.0}}{}}
\item{\cventry{Geneseo, NY}{B.Sc. in Biochemistry, minor in Biomathematics}{State University of New York at Geneseo}{May 2015}{\textit{GPA: 3.660/4.0}}{}}

\end{itemize}

%-------------------------------------------------------------------------------------------
\section{Awards}

\begin{itemize}
  
\item \cventry{}{Received a potentially fundable score of 30 applying for NLM F31 fellowship, ultimately not funded}{Finalist, Ruth L. Kirschstein National Research Service Award (NRSA) Individual Predoctoral Fellowship}{2020}{}{}

\item \cventry{}{Competitive departmental T32 fellowship offered through Biomedical Sciences Graduate Program at Ohio State}{Systems and Integrative Biology Training Program Fellow}{2017-2019}{}{}

\item \cventry{}{Travel stipend for student attendees of American Society of Mass Spectrometry annual conference}{American Society of Mass Spectrometry Travel Award}{2018 and 2019}{}{}

\item \cventry{}{Travel stipend for attendees of Northeastern University May Institute of mass spectrometry}{May Institute Travel Award}{2018}{}{}

\item \cventry{}{Departmental T15 fellowship offered through Biomedical informatics department at Ohio State}{Multi-Modeling and Integrative Data Analytics Training Program fellow}{2016-2017}{}{}

\item \cventry{}{Awarded during NIAMS Scientific Planning Retreat}{Best Poster Award}{2016}{}{}

\item \cventry{}{Funded by American Society for Pharmacology and Experimental Therapeutics}{Summer Undergraduate Research Fellowship at University at Buffalo}{2014}{}{}

  \end{itemize}
% -------------------------------------------------------------------------------------------
\section{Publications}

\begin{enumerate}

\item \textbf{Patt, A.}; Demoret, B.; Stets, C.; Bill, K.-L.; Smith, P.; Vijay, A.; Patterson, A.; Hays, J.; Hoang, M.; Chen, J.L.; Mathé, E.A. \emph{MDM2-Dependent Rewiring of Metabolomic and Lipidomic Profiles in Dedifferentiated Liposarcoma Models.} Cancers 2020, 12, 2157.

\item \emph{An OpenData portal to share COVID-19 drug repurposing data in real time.} Kyle R. Brimacombe, Tongan Zhao, Richard T. Eastman, Xin Hu, Ke Wang, Mark Backus, Bolormaa Baljinnyam, Catherine Z. Chen, Lu Chen, Tara Eicher, Marc Ferrer, Ying Fu, Kirill Gorshkov, Hui Guo, Quinlin M. Hanson, Zina Itkin, Stephen C. Kales, Carleen Klumpp-Thomas, Emily M. Lee, Sam Michael, Tim Mierzwa, \textbf{Andrew Patt}, Manisha Pradhan, Alex Renn, Paul Shinn, Jonathan H. Shrimp, Amit Viraktamath, Kelli M. Wilson, Miao Xu, Alexey V. Zakharov, Wei Zhu, Wei Zheng, Anton Simeonov, Ewy A. Mathé, Donald C. Lo, Matthew D. Hall, Min Shen. bioRxiv 2020.06.04.135046; doi: https://doi.org/10.1101/2020.06.04.135046
  
\item Eicher, T.; Kinnebrew, G.; \textbf{Patt, A.}; Spencer, K.; Ying, K.; Ma, Q.; Machiraju, R.; Mathe, E.A. \emph{Metabolomics and Multi-Omics Integration: A Survey of Computational Methods and Resources}. Metabolites 2020, 5, 15.

\item \textbf{Patt, A.}; Eicher, T.; Kautto, E.; Machiraju, R.; Mathe, E.A.; Zhang, Y. \emph{Challenges in proteogenomics: a comparison of analysis methods with the case study of the DREAM Proteogenomics Sub-Challenge}. BMC Bioinformatics 2019;20(Suppl 24):669.

\item Zhang, B.; Hu, S.; Baskin, E.; \textbf{Patt, A.}; Siddiqui, J.K.; Mathe, E.A. \emph{RaMP: A Comprehensive Relational Database of Metabolomics Pathways for Pathway Enrichment Analysis of Genes and Metabolites}. Metabolites 2018, 8, 16.

\item \textbf{Patt A.;} Siddiqui J.; Zhang B.; Mathe E.A. (2019) \emph{Integration of Metabolomics and Transcriptomics to Identify Gene-Metabolite Relationships Specific to Phenotype.} In: Haznadar M. (eds) Cancer Metabolism. Methods in Molecular Biology, vol 1928. Humana Press, New York, NY
\end{enumerate}
%-------------------------------------------------------------------------------------------
\section{Presentations}
\begin{itemize}

\item \cventry{Online}{Poster}{Metabolomics Society Annual Meeting}{November 2020}{}{\emph{Improving pathway analysis of lipidomic and metabolomic data through comprehensive functional annotation and network approaches}}
  
\item \cventry{Ann Arbor, MI (virtual)}{15 minute talk}{Metabolomics Association of North America Annual Meeting}{September 2020}{}{\emph{Improving pathway analysis of lipidomic and metabolomic data through comprehensive functional annotation and network approaches}}

\item \cventry{Houston, TX (virtual)}{Poster}{American Society for Mass Spectrometry}{June 2020}{}{\emph{MDM2 copy number aberrations alter ceramide glycosylation in liposarcoma tumors, impacting drug response}}

\item \cventry{Columbus, OH}{Poster}{Wexner Medical Center Trainee Research Day}{April 2020}{}{\emph{MDM2 copy number aberrations alter ceramide glycosylation in liposarcoma tumors, impacting drug response}}

  \item \cventry{Columbus, OH}{Poster}{Ohio Mass Spectrometry and Metabolomics Symposium}{October 2019}{}{\emph{MDM2 copy number aberrations alter ceramide glycosylation in liposarcoma tumors, impacting drug response}}
  
\item \cventry{Columbus, OH}{20 minute talk}{International Conference on Intelligent Biology and Medicine}{June 2019}{}{\emph{Challenges in proteogenomics: a comparison of analysis methods with the case study of the DREAM Proteogenomics Sub-Challenge}}

\item \cventry{Atlanta, GA}{Poster}{American Society for Mass Spectrometry}{June 2019}{}{\emph{MDM2 Alterations Reprogram the Metabolic Functioning of Liposarcoma Cell Lines}}

\item \cventry{Columbus, OH}{Poster}{Ohio State University Cancer Care Center Annual Meeting}{May 2019}{}{\emph{MDM2 Alterations Reprogram the Metabolic Functioning of Liposarcoma Cell Lines}}

\item \cventry{San Diego, CA}{Poster}{American Society for Mass Spectrometry}{June 2018}{}{\emph{MDM2 alterations reprogram the metabolic functioning of liposarcoma cell lines}}

  \item \cventry{Columbus, OH}{Poster}{Ohio Mass Spectrometry and Metabolomics Symposium}{April 2018}{}{\emph{MDM2 Alterations Reprogram the Metabolic Functioning of Liposarcoma Cell Lines}}
  
  \item \cventry{La Jolla, CA}{5-minute talk}{National Library of Medicine Informatics Training Conference}{June 2017}{}{\emph{MDM2 alterations reprogram the metabolic functioning of liposarcoma cell lines}}

\item \cventry{Columbus, OH}{Poster}{Wexner Medical Center Trainee Research Day}{April 2017}{}{\emph{MDM2 alterations reprogram the metabolic functioning of liposarcoma cell lines}}

\item \cventry{Bethesda, MD}{Poster}{NIAMS Scientific Planning Retreat}{April 2016}{}{\emph{Deep Resequencing of Juvenile Arthritis Patients Identifies Rare Variant Associations}}

\item \cventry{Bethesda, MD}{Poster}{Postbac Poster Day}{April 2016}{}{\emph{Deep Resequencing of Juvenile Arthritis Patients Identifies Rare Variant Associations}}

\item \cventry{Geneseo, NY}{Poster}{SUNY Geneseo GREAT Day}{April 2015}{}{\emph{Sex Differences in MCT1 Tissue Expression}}
  
\item \cventry{Buffalo, NY}{20 minute talk and poster}{University at Buffalo Summer Research Symposium}{August 2014}{}{\emph{Sex Differences in MCT1 Tissue Expression}}
\end{itemize}

%-------------------------------------------------------------------------------------------
\section{Research Experience}
\begin{itemize}
\item \cventry{Rockville, MD (remote position)}{Research Associate}{National Center for Advancing Translational Sciences}{March 2020-Present}{}{Working with Dr. Ewy Mathe in the Bioinformatics core at the National Center for Advancing Translational Sciences. Contracted through Axle Informatics. \\•Continued projects from OSU including providing guidance for updates of RaMP database/R package.\\•Developing a method for improved functional enrichment analysis of metabolomic data (including pathway analysis) using biological pathway and chemical structure knowledge graphs.\\•Contributing to identifying new, publicly available 'omics data generated in the study of COVID-19. See data portal: https://opendata.ncats.nih.gov/covid19/omics}
\item \cventry{Columbus, OH}{Graduate Research Assistant}{The Ohio State University, Biomedical Informatics Department}{August 2016-Present}{}{Working with Dr. Ewy Mathe in the Biomedical Informatics department through the Biomedical Sciences Graduate Program.\\•Investigating metabolomic and transcriptomic differences between dedifferentiated liposarcoma tumors based on copy number of the MDM2 oncogene.\\•Assisted in developing RaMP metabolic pathways database/R package and Rshiny app.\\•Worked primarily with the R programming language as well as Bash, MySQL and Python.\\•Ran memory-intensive scripts on the Ohio Supercomputing Cluster.}
\item \cventry{Bethesda, MD}{Postbaccalaureate Fellow}{National Institute of Arthritis, Musculoskeletal and Skin Diseases}{July 2015-July 2016}{}{Worked in the lab of Dr. Michael Ombrello through NIH postbaccalaureate program. \\•Searched for ultrarare genetic variants associated with Systemic Juvenile Idiopathic Arthritis in a targeted resequencing project. \\•Used Illumina sequencing kits as well as GATK and annovar softwares. \\•Worked with R, Excel and Bash for data sorting, manipulation and analysis.\\•Gained experience working with supercomputing clusters.}
\item \cventry{Buffalo, NY}{Summer Undergraduate Research Fellow}{University at Buffalo, Pharmaceutical Sciences Department}{May 2014-August 2014}{}{Summer undergraduate research fellowship funded by the American Society of Pharmacology and Experimental Therapeutics. Worked with Dr. Marilyn Morris and Dr. Melanie Felmlee in the Pharmaceutical Science department at University at Buffalo.\\•Performed RNA extraction/qPCR using rat tissue\\•Recieved weekly coaching on oral/poster presentations of research\\•Discovered a significantly lower level of monocarboxylate transporter expression in the livers of female rats per gram of tissue, potentially leading to differences in metabolism of monocarboxylate compounds.}
\item \cventry{Geneseo, NY}{Undergraduate Research Assistant}{State University of New York at Geneseo, Biology Department}{September 2013-May 2015}{}{•Worked in the Biomathematics research interest group of Dr. Gregg Hartvigsen, learning basic biological network modeling in the R programming language.\\•Worked with Dr. Isidro Bosch identifying species of starfish larvae samples using targeted sequencing.}  
\end{itemize}
%-------------------------------------------------------------------------------------------
\section{Work Experience}

\vspace{1pt}

\begin{itemize}
\item \cventry{Schenectady, NY}{Lab Assistant}{Pace Analytical}{June 2013-January 2014}{}{•Organized/analyzed chemical and supply purchasing data\\•Organized stock room\\•Performed sample delivery\\•General lab support}
\item \cventry{Geneseo, NY}{Assistant Lab Instructor}{SUNY Geneseo}{January 2013-May 2013}{}{Instructed single-semester human biology lab, aided students with experiments and lab reports.}
\item \cventry{Schenectady, NY}{Adult Learn-to-Row Coach}{Aqueduct Rowing Club}{May 2012-August 2012}{}{Coached two 5-week sessions of an introductory rowing course. Required to devise and execute practice routines to instruct novice adult sweep rowers.}
\end{itemize}
% ------------------------------------------------------------------------------------------
\section{Volunteering and Leadership}

\begin{itemize}
\item  \cventry{Columbus, Ohio}{Organizer}{Friday Afternoon Bioinformatics Workshops}{January 2019-June 2019}{}{Organized and presented at a series of seminars that provided overviews of useful software tools for bioinformaticians, including Git, Bash, Docker, Tidyverse, Emacs and mzMine}
\item  \cventry{Bethesda, MD}{Nature Center Greeter}{Locust Grove Nature Center}{August 2015-November 2015}{}{Helped orient visitors, aided in park events, tracked park usage and maintained trail and visitor center on weekends.}
\item  \cventry{Albany, NY}{WAVE Volunteer}{Wadeable Assessment by Volunteer Evaluation Program}{May 2012-August 2012}{}{Collected samples of Benthic Macroinvertebrates and performed an amateur assessment of stream health from various wadeable stream habitats to aid in a large scale evaluation of freshwater ecosystems health in upstate New York.}
\end{itemize}
%------------------------------------------------------------------------------------------
\section{Relevant Coursework}
\subsection{Graduate}

\textbf{Northeastern May Institute}
\hfill
\textbf{May 2018}

•Computation and Statistics for Mass Spectrometry and Proteomics (week-long workshop)

\vspace{5pt}

\textbf{The Ohio State University}
\hfill
\textbf{August 2016-October 2018}

•Introduction to Biomedical Informatics
\\•Analysis and Applications of Genome-Scale Data
\\•Seminars in Bioinformatics
\\•Data Structures and Algorithms
\\•Probability for statistical inference
\\•Metabolomics, Principles and Practice (audited)

\vspace{5pt}

\textbf{Foundation for Advanced Education in the Sciences}
\hfill
\textbf{September 2015-May 2016}

•Practical Bioinformatics\\•Introduction to Python

\vspace{5pt}

\subsection{Undergraduate}

\textbf{State University of New York at Geneseo}
\hfill
\textbf{September 2011-May 2015}

•Lab: Biochemistry, Molecular Techniques, Organic Chemistry, Biology, Plant Physiology, Analytical Physics
\\•Biology: Microbiology, Cell Biology, Genetics, Ecology
\\•Chemistry: Organic Chemistry, Biochemistry, Physical Chemistry
\\•Math/Physics: Calculus 1,2,3, Linear Algebra, Probability, Differential Equations, Biostatistics, Modeling Biological
Systems, Analytical Physics

\vspace{1pt}
% Publications from a BibTeX file without multibib
%  for numerical labels: \renewcommand{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}% CONSIDER MERGING WITH PREAMBLE PART
%  to redefine the heading string ("Publications"): \renewcommand{\refname}{Articles}

\section{Relevant Skills}
\begin{itemize}
\item \textbf{Programming Languages:} R, Python, Bash, MySQL

\item \textbf{Mass Spec Data Analysis:} mzMine, XCMS, LipidMatch, Metaboanalyst, RaMP, MS-DIAL, Excel

\item \textbf{Genomic Data Analysis:} Picard, GATK, Annovar

\item \textbf{Wet Lab:} RNA extraction and purification (Qiagen),
  Illumina MiSeq library preparation (Nextera Rapid Capture Custom
  Enrichment kits)
  
\item \textbf{Development Tools:} Emacs, Linux, RStudio,
  Git/Github/Gitlab, RShiny, RMarkdown, Tidyverse, Devtools

\item \textbf{Public Speaking:} Throughout my undergraduate and
  graduate careers I have given a large number of oral presentations
  describing my work, from small group settings to major international
  conferences.

\item \textbf{Technical Writing:} I have been the primary
  author/coauthor on three peer-reviewed manuscripts, as well as
  secondary author on three more. I have also been the primary
  submitter on several grant proposals, including a proposal for a
  departmental T32 award which funded two years of my tuition/stipend
  at OSU, and an F31 proposal which recieved a potentially fundable
  score of 30.
  
\end{itemize}


\section{Society Memberships}

\begin{itemize}
  \item \textbf{Metabolomics Society:} 2020-Present
  \item \textbf{Metabolomics Association of North America:} 2020-Present
  \item \textbf{American Society for Mass Spectrometry:} 2018-Present
  \end{itemize}
% Publications from a BibTeX file using the multibib package
%\section{Publications}
%\nocitebook{book1,book2}
%\bibliographystylebook{plain}
%\bibliographybook{publications}                   % 'publications' is the name of a BibTeX file
%\nocitemisc{misc1,misc2,misc3}
%\bibliographystylemisc{plain}
%\bibliographymisc{publications}                   % 'publications' is the name of a BibTeX file

%-----       letter       ---------------------------------------------------------

\end{document}


%% end of file `template.tex'.
